tests/pos/extmethods.scala

```diff
@@ -5,3 +5,13 @@ class T[A, This <: That1[A]](val x: Int) extends AnyVal {
   final def loop(x: This, cnt: Int): Int = loop(x, cnt + 1)
   def const[B](): Boolean = return true
 }
+
+final class TraversableOnceOps[+A](val collection: TraversableOnce[A]) extends AnyVal {
+  def reduceLeftOption[B >: A](op: (B, A) => B): Option[B] =
+    if (collection.isEmpty) None else Some(collection.reduceLeft[B](op))
+}
+
+class Foo[+A <: AnyRef](val xs: List[A]) extends AnyVal {
+  def baz[B >: A](x: B): List[B] = ???
+}
+
```

Comment:
Instead of adding these tests here, you should move `tests/pending/pos/t6482.scala` and `tests/pending/pos/t7022.scala` to `tests/pos/`.